android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -349,11 +370,14 @@ public synchronized void onCompletion(MediaPlayer mp) {
       @Override
       public synchronized boolean onError(MediaPlayer mp, int what, int extra) {
         setOnPlay(false, key);
-        if (callbackWasCalled) return true;
+        if (callbackWasCalled) {
+          return true;
+        }
+        cancelTimer(key);
         callbackWasCalled = true;
         try {
-          callback.invoke(true);
-        } catch (Exception e) {
+          callback.invoke(PLAY_RESULT_SUCCESS);
```

Comment:
Should this be `PLAY_RESUT_SUCCESS` in onError?

Reply:
changed to `PLAY_RESULT_FAILURE` in commit [#0423746](https://github.com/tomtom-international/react-native-sound/pull/5/commits/04237469ad387c6797d4f7cb7eff742a99fb60ef)